web/src/hash_util.ts

```diff
@@ -288,7 +288,7 @@ export function validate_group_settings_hash(hash: string): string {
 
 export function decode_stream_topic_from_url(
     url_str: string,
-): {stream_name: string; topic_name?: string} | null {
+): {stream_id: string; topic_name?: string} | null {
```

Comment:
Call this `stream_id_string` or return the integer, whichever seems cleaner for how this is used.

Reply:
Returning a number seems good